src/constants.rs

```diff
@@ -187,3 +187,100 @@ impl TryFrom<u32> for DelimType {
         }
     }
 }
+
+/// Valid child types of Document elements
+pub const DOCUMENT_CHILDREN: &[NodeType] = &[
```

Comment:
Is there any value in making this a set instead of an array? If you're only doing membership queries, then it could better describe the intent.

Reply:
Replaced with lazy static HashSets